packages/lit-dev-content/site/tutorials/content/first-component/05.md

```diff
@@ -0,0 +1,66 @@
+In this section you will add scoped styles that are the same for every instance
+of the component.
+
+Define scoped styles in the `static styles` class field using the `css` tag
+function.
+
+{% switchable-sample %}
+
+```ts
+// word-viewer.ts
+
+  static styles = css`
+    :host {
+      background-color: white;
+      color: violet;
+      cursor: pointer;
+      display: block;
+    }
+    pre {
+      padding: 0.2em;
+    }
+  `;
+
+```
+
+```js
+// word-viewer.js
+
+  static styles = css`
+    :host {
+      background-color: white;
+      color: violet;
+      cursor: pointer;
+      display: block;
+    }
+    pre {
+      padding: 0.2em;
+    }
+  `;
+
+```
+
+{% endswitchable-sample %}
+
+You've defined two selectors here, `:host` and `pre`. `pre` behaves like you may
+expect if you have prior familiarity with CSS, and will add padding to the `pre`
+element that is rendered from the template.
+
+[`:host`](https://developer.mozilla.org/en-US/docs/Web/CSS/:host) is a selector
+for styling the component itself. The element that owns, or "hosts" a [shadow
+tree](/docs/components/styles/#shadow-dom) is called the _host
+element_.
+
+These styles are strongly encapsulated, so a `pre` element outside the
+component's [shadow DOM](/docs/components/styles/#shadow-dom)) will not be
+selected.
+
+
+<litdev-aside type="info">
+
+Extra credit
+
+Use the [`:host()`](/docs/components/styles/#host) selector so that adding a
+`green` class on the `word-viewer` results in green text.
+
+</litdev-aside>
```

Comment:
I've left my "Extra credit" prompt outside the aside. I'm not sure which way we want to go but if you do keep the aside, maybe `type="positive"` would be more appropriate? A checkmark feels more appropriate to me than an "i" icon.